docs/cli.md

```diff
@@ -616,23 +616,25 @@ viam logout
 
 ### `module`
 
-The `module` command allows to you to manage custom {{< glossary_tooltip term_id="module" text="modules" >}}
+The `module` command allows to you to manage custom {{< glossary_tooltip term_id="module" text="modules" >}}.
 This includes:
 
-- Creating a new custom {{< glossary_tooltip term_id="resource" text="modular resource" >}}
+- Generating stub files for a new module
+- Creating metadata for a new custom {{< glossary_tooltip term_id="resource" text="modular resource" >}}
 - Uploading a new module to the [Viam registry](https://app.viam.com/registry)
 - Uploading a new version of your module to the [Viam registry](https://app.viam.com/registry)
 - Updating an existing module in the Viam registry
 - Updating a module's metadata file based on models it provides
 - Building your module for different architectures using cloud runners
 - Building a module locally and running it on a target device. Rebuilding & restarting if already running.
 
-See [Upload a custom module](/how-tos/create-module/#upload-your-module-to-the-modular-resource-registry) and [Update an existing module](/how-tos/manage-modules/#update-an-existing-module) for a detailed walkthrough of the `viam module` commands.
+See [Upload a custom module](/how-tos/upload-module/) and [Update an existing module](/how-tos/manage-modules/#update-an-existing-module) for a detailed walkthrough of the `viam module` commands.
 
 If you update and release your module as part of a continuous integration (CI) workflow, you can also
 [automatically upload new versions of your module on release](/how-tos/manage-modules/#update-an-existing-module-using-a-github-action) using a GitHub Action.
 
 ```sh {class="command-line" data-prompt="$"}
+viam module generate --resource-type=<component|service> --resource-subtype=<subtype>
```

Comment:
the args are optional, and the preferred path is to actually not include them. i would remove them from this particular example

Reply:
I believe this section is meant to illustrate all possible options, whereas the next section (line 652) is *examples*. I will add a note to the table below suggesting they not use the options. Also note that, in a different currently-open PR, we do guide people through creating a module using this command, and we do not use options.

Reply:
We can make an exception if we're that opinionated about it  
```suggestion
viam module generate
```

Reply:
Okay. I'll leave the notes I added in the table to explain this, unless you think that's overkill now that this is coming out.

Reply:
Thanks. The CLI flags are still not fully baked yet so I don't want people relying on them yet